cmd/yurt-controller/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
all Dockerfile are placed under `hack/dockerfiles` dir.

Reply:
This is only temporary and will be transferred when the processing makes build